.github/workflows/azure-static-web-apps-wonderful-sea-0ff5ee003.yml

```diff
@@ -0,0 +1,45 @@
+name: Azure Static Web Apps CI/CD
+
+on:
+  push:
+    branches:
+      - frontend-new-strcutre-adminsite
+  pull_request:
+    types: [opened, synchronize, reopened, closed]
+    branches:
+      - frontend-new-strcutre-adminsite

```

Comment:
I'd also change this to main so it will will create a preview branch when a PR is created where the target branch is main